src/Mvc/Mvc.DataAnnotations/src/DataAnnotationsMetadataProvider.cs

```diff
@@ -27,9 +27,6 @@ internal class DataAnnotationsMetadataProvider :
     private const string NullableAttributeFullTypeName = "System.Runtime.CompilerServices.NullableAttribute";
     private const string NullableFlagsFieldName = "NullableFlags";
```

Comment:
Can this also be removed?

Reply:
It's still used in this method ([ref](https://github.com/dotnet/aspnetcore/blob/206321282e149074da204366e6590fa0590664b7/src/Mvc/Mvc.DataAnnotations/src/DataAnnotationsMetadataProvider.cs#L476-L505)) which is used [here](https://github.com/dotnet/aspnetcore/blob/206321282e149074da204366e6590fa0590664b7/src/Mvc/Mvc.DataAnnotations/src/DataAnnotationsMetadataProvider.cs#L364-L370) where we don't have access to the `PropertyInfo` and only the attributes on it.

Reply:
I would say it might be ok to remove that code now. We'll have enough time in this release to know if we break someone (super unlikely)